core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/util/CsvUtil.java

```diff
@@ -69,4 +95,22 @@ public static <T> byte[] itemsToCsvByteArray(Collection<T> items, BiConsumer<CSV
             return ArrayUtils.EMPTY_BYTE_ARRAY;
         }
     }
+
+    public static List<CSVRecord> readCsvItems(InputStream inputStream, String[] headersArray) {
+        List<CSVRecord> csvRecords = new ArrayList<>();
+        if (inputStream == null) {
+            return csvRecords;
+        }
+        CSVFormat csvFormat = CSVFormat.DEFAULT.withHeader(headersArray);
+        Reader reader = new BufferedReader(new InputStreamReader(inputStream));
+        try {
+            final Iterable<CSVRecord> records = csvFormat.parse(reader);
+            for (CSVRecord linkRecord : records) {
+                csvRecords.add(linkRecord);
+            }
+        } catch (IOException e) {
+            LOG.error("Failed to read CSV item form jcr.", e);
```

Comment:
(cosmetic) Why CSV is all captions but jcr is not? Theu are both abbreviations ))
Also, please carefully scan through all the comments, logs and javadocs for uniform spelling of words like JCR, CSV, etc. I have seen either "csv", "Csv", and "CSV" in this PR)